components/agenda/src/Agenda.svelte

```diff
@@ -248,29 +248,26 @@
   // #region Data Flow
 
   // Accept either comma-separated string, or array.
-  $: calendarIDs = (() => {
+  let calendarIDs: string[] = [];
+  $: (calendarIDs = setCalendarIDs()), internalProps, calendar_id;
```

Comment:
I think you should only need to reference `internalProps.calendar_id` here
```suggestion
  $: (calendarIDs = setCalendarIDs()), internalProps.calendar_id;
```